docs/data-sources/infoblox_host_record.md

```diff
@@ -0,0 +1,110 @@
+# Host-record Data Source
+
+Use the `infoblox_host_record` data source to retrieve the following information for a Host-Record if any, which is managed by a NIOS server:
+
+* `dns_view`: the DNS view which the record's zone belongs to. Example: `default`
+* `fqdn`: the fully qualified domain name which the IP address is assigned to. `blues.test.com`
+* `ip4_addr`: the IPv4 address associated with the Host-record. Example: `10.0.0.32`
+* `ip6_addr`: the IPv6 address associated with the Host-record. Example: `2001:1890:1959:2710::32`
+* `mac_addr`: the MAC address associated with the Host-record. Example: `aa:bb:cc:dd:ee:11`
+* `zone`: the zone that contains the record in the specified DNS view. Example: `test.com`.
+* `ttl`: the "time to live" value of the record, in seconds. Example: `1800`.
+* `duid`: the DHCP Unique Identifier of the record. Example: `34:df:37:1a:d9:7f`.
+* `enable_dns`: the flag to enable or disable the DNS record. Example: `true`.
+* `enable_dhcp`: the flag to enable or disable the DHCP record. Example: `true`.
+* `comment`: the description of the record. This is a regular comment. Example: `Temporary A-record`.
+* `ext_attrs`: the set of extensible attributes of the record, if any. The content is formatted as string of JSON map. Example: `"{\"TestEA\":56,\"TestEA1\":\"kickoff\"}"`
+
+As there is new feature filters , the previous usage of combination of DNS view, IPv4 address and FQDN, has been removed.
+
+For usage of filters, add the fields as keys and appropriate values to be passed to the keys like `name`, `view` corresponding to object.
+From the below list of supported arguments for filters,  use only the searchable fields for retriving the matching records.
+
```

Comment:
We can use the same format as in doc:
"To retrieve information about host records that match the specified filters, use the filters argument and specify the parameters mentioned in the below table. These are the searchable parameters of the corresponding object in Infoblox NIOS WAPI. If you do not specify any parameter, the data source retrieves information about all host records in the NIOS Grid.

The following table describes the parameters you can define in an infoblox_host_record data source block:"